joomla/joomla-cms
# com_contact. Rm unused order_up, order_dn

`order_up` and `order_dn` are not used in Joomla core. Garbage from Joomla 1.6 or so.

### Summary of Changes
Removed lines from com_contact list view.

```
// Preprocess the list of items to find ordering divisions.
// @todo: Complete the ordering stuff with nested sets
foreach ($this->items as &$item) {
 $item->order_up = true;
 $item->order_dn = true;
}
```

### Testing Instructions
- Check the Joomla core if you find somewhere a code place where properties `order_up` or `order_dn` are used. You won't find one.
- Check that `com_contact` list view in Joomla backend works like before when these lines are removed.
- @Kubik-Rubik [confirmed in german Joomla forum that these lines are garbage](https://forum.joomla.de/thread/20144-item-order-up-item-order-dn-in-komponente-was-ist-das/?postID=146525#post146525) from Joomla 1.6 or so.

### Actual result BEFORE applying this Pull Request
- Unused garbage in file.

### Expected result AFTER applying this Pull Request
- No functional changes. Garbage removed.

### Link to documentations
Please select:
